xml/System.Numerics/Complex.xml

```diff
@@ -1832,6 +1832,45 @@
         <altmember cref="F:System.Numerics.Complex.Zero" />
       </Docs>
     </Member>
+    <MemberGroup MemberName="op_Addition">
+      <AssemblyInfo>
+        <AssemblyName>System.Runtime.Numerics</AssemblyName>
+        <AssemblyVersion>4.1.1.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Docs>
+        <summary>Adds a specified number to another specified number.</summary>
```

Comment:
Maybe calling out that this adds a 'real' value and 'complex' value would be good?

Maybe something like:
> Adds a specified real number to a specified complex number.

Reply:
It's the member group description. It wraps all the additions overloads: complex+complex, complex+double, double+complex.

Reply:
Ah, I had missed this was the member group description.

Reply:
How about:

```suggestion
        <summary>Adds a specified number to another specified number, where at least one of them is a complex number, and the other could be a double-precision real number.</summary>
```